typings/morphdom.d.ts

```diff
@@ -0,0 +1,23 @@
+declare module 'morphdom' {
+	interface MorphDomOptions {
+		getNodeKey?: (node: Node) => any,
+		onBeforeNodeAdded?: (node: Node) => Node,
+		onNodeAdded?: (node: Node) => Node,
+		onBeforeElUpdated?: (fromEl: HTMLElement, toEl: HTMLElement) => boolean,
+		onElUpdated?: (el: HTMLElement) => void,
+		onBeforeNodeDiscarded?: (node: Node) => boolean,
+		onNodeDiscarded?: (node: Node) => void,
+		onBeforeElChildrenUpdated?: (fromEl: HTMLElement, toEl: HTMLElement) => boolean,
+		childrenOnly?: boolean
+	}
+
+	namespace morphdom {}
+
+	function morphdom(
+		fromNode: Node,
+		toNode: Node | string,
+		options?: MorphDomOptions
+	): void;
+
+	export = morphdom;
```

Comment:
I had to change this line to `export default morphdom;` to get this to work in my project.

Reply:
Does that need to be updated in this PR?